clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -220,7 +220,8 @@ def _get_pods_by_ip(self) -> Mapping[str, List[KubernetesPod]]:
         all_pods = self._core_api.list_pod_for_all_namespaces().items
         pods_by_ip: Mapping[str, List[KubernetesPod]] = defaultdict(list)
         for pod in all_pods:
-            if pod.status.phase == 'Running' and pod.status.host_ip in self._nodes_by_ip:
+            if (pod.status.phase == 'Running' or pod.status.phase == 'Pending') \
```

Comment:
could we make this a set?  e.g.
```KUBERNETES_SCHEDULED_PHASES = {'Pending', 'Running'}```
and then we can just check `if pod.status.phase in KUBERNETES_SCHEDULED_PHASES`